.github/workflows/codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+    - "common/config.py"
```

Comment:
Could this be somehow added here instead?

https://github.com/RedHatInsights/vulnerability-engine/blob/master/.coveragerc